src/Spice86.Core/Emulator/CPU/InstructionsImpl/Instructions16.cs

```diff
@@ -526,6 +528,42 @@ public class Instructions16 : Instructions16Or32 {
         Cpu.Out16(State.DX, State.AX);
     }
 
+    public override void Enter() {
```

Comment:
I think this is not correct because stackSize is 16 for 16bits instructions and 32 for 32bit.
For the 32bits I would rather use the algorithm descried here: https://xem.github.io/minix86/manual/intel-x86-and-64-manual-vol1/o_7281d5ea06a5b67a-159.html
And adapt it with BP instead of EBP and SP instead of ESP for 16 bits.

Reply:
Done.